### PR TITLE
qa_crowbarsetup.sh: deploy hawk to clusters

### PR DESCRIPTION
**Pull request by aspiers**

### CODE REVIEW

**Comment by bmwiedemann** [comment]
LGTM

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1484,10 +1484,12 @@ function hacloud_configure_cluster_members()
 
     local nodes=`printf "\"%s\"," $@`
     nodes="[ ${nodes%,} ]"
-    proposal_modify_value pacemaker "$clustername" \
-        "['deployment']['pacemaker']['elements']['pacemaker-cluster-member']" "[]" "||="
-    proposal_modify_value pacemaker "$clustername" \
-        "['deployment']['pacemaker']['elements']['pacemaker-cluster-member']" "$nodes" "+="
+    for role in pacemaker-cluster-member hawk-server; do
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: please add a line above: local role

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: @jdsn Done, thanks for reminder!

**Comment by bmwiedemann** [comment]
:+1: if it is tested

**Comment by jdsn** [comment]
:+1: if testrun succeded

**Comment by aspiers** [comment]
It works :)

**Pull request merged**